src/cockpit/channels/dbus.py

```diff
@@ -220,9 +220,9 @@ async def do_call(self, call, message):
         # We have to figure out the signature of the call.  Either we got told it:
         signature = message.get('type')
 
-        # ... or all of our arguments are string (and we can guess):
-        if signature is None and all(isinstance(arg, str) for arg in args):
```

Comment:
Pretty sure it's caused by this change.  Before, `GetAll` was a method where all parameters were strings, so we skipped introspection.

I suspect this is a case of uncovering a bug rather than making a new one, but it's still a regression...